src/APICalls.js

```diff
@@ -0,0 +1,124 @@
+import axios from "axios";
+
+const kBoardBaseUrl = "https://inspiration-board-backend-t6x0.onrender.com/boards";
```

Comment:
Prefer a single shared base url for the api host (https://inspiration-board-backend-t6x0.onrender.com) rather than making one base url value for boards and one for cards. Of course you can still have a boards and card value (to avoid typos), but calculate it using a value that holds the common api host part of the url.